src/server/services/wms/qgswmsrestorer.cpp

```diff
@@ -138,10 +140,33 @@ QgsLayerRestorer::~QgsLayerRestorer()
   }
 }
 
+QgsProjectRestorer::QgsProjectRestorer( const QgsProject *project )
```

Comment:
```suggestion
QgsProjectRestorer::QgsProjectRestorer( QgsProject *project )
```

A project restorer says in its name that it will modify the project. So why suggest it would be const? :-)